code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -617,15 +617,21 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 		return TOPIC_REFRESH
 
 	else if(href_list["organs"])
-		var/organ_name = input(user, "Which internal function do you want to change?") as null|anything in list(BP_HEART, BP_EYES)
+		var/organ_name = input(user, "Which internal function do you want to change?") as null|anything in list("Heart", "Eyes", "Lungs", "Liver", "Kidneys")

```

Comment:
Any reason we ain't just using the defines for all of 'em? Not using defines seems like a bit of a regression

Reply:
Because these are display names and we want them capitalized

Reply:
Thanks for clearing that up, makes sense